web/controllers/weekly_info_controller.ex

```diff
@@ -40,10 +54,12 @@ defmodule Syndeo.WeeklyInfoController do
     |> redirect(to: attendee_weekly_info_path(conn, :index, attendee_id))
   end
 
-  defp render_index(conn, %Attendee{}=attendee, changeset) do
+  defp render_index(conn, %Attendee{}=attendee, meal, changeset) do
     conn
     |> assign(:attendee, attendee)
     |> assign(:weekly_info, attendee.weekly_info)
+    |> assign(:meal, meal)
+    |> assign(:no_wednesday_meal, is_nil(meal))
```

Comment:
I think I would remove this.  I'd add a virtual field to the `Meal`, something like "available_for_reservations". By default, all the ones in your DB could be true and in your "null object", set it to false. Then in your template, you can check for that boolean and display accordingly.

That makes it a bit more abstract so it's more flexible in the future.